modules/ossm-tutorial-bookinfo-removing.adoc

```diff
@@ -45,7 +45,7 @@ $ oc delete project bookinfo
 
 . Click *Operators* -> *Installed Operators*.
 
-. Click the *Project* menu and choose `openshift-operators` from the list.
+. Click the *Project* menu and choose `istio-system` from the list.
```

Comment:
Good catch!  I can't believe someone didn't catch this before now.  =(

Reply:
But now it's fixed, so flip that frown =)